bugbug/similarity.py

```diff
@@ -140,6 +147,9 @@ def evaluation(self):
             and "dupeme" not in bug["keywords"]
         )
 
+        if end_to_end:
+            duplicatemodel = DuplicateModel.load("duplicatemodel")
```

Comment:
In a follow-up, we should remove this from `evaluation` and make it run (optionally) as part of `get_similar_bugs`.

Reply:
Yeah, I'll open a new PR for that.